hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/om/om.tsx

```diff
@@ -1045,10 +1077,11 @@ export class Om extends React.Component<Record<string, object>, IOmdbInsightsSta
     const paginationConfig: PaginationConfig = {
       pageSize:this.state.DEFAULT_LIMIT,
       defaultPageSize: this.state.DEFAULT_LIMIT,
-      pageSizeOptions: ['10', '20', '30', '50'],
-      showSizeChanger: true,
+      pageSizeOptions: ['10', '30', '50', '100'],
+      showSizeChanger: this.state.nextClickable ? true : false,
       onShowSizeChange: this.onShowSizeChange,
-      itemRender: this.itemRender
+      itemRender: this.itemRender,
+      total: this.state.pageDisplayCount - 1
```

Comment:
What is `total` here?

Reply:
@dombizita 
This is antd pagination property if we remove this attribute
We will get such UI with no Pagination or we could not go to previous pages.

![image](https://github.com/apache/ozone/assets/112169209/bba8b5b9-375e-4ac7-8dee-0999981a0212)